torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -88,8 +89,21 @@ def validate_args_and_maybe_create_graph_inputs(
             if manually_set_subgraph_inputs:
                 new_proxy = tracer.create_graph_input(a.as_proxy().node.name)
                 example_value = a.as_proxy().node.meta["example_value"]
+
+                options = {}
+                if is_traceable_wrapper_subclass(example_value):
```

Comment:
`is_traceable_wrapper_subclass` is not the useful qualifier, its absence of a source. What I would urge you to do instead is lift this up as input.